Add private _export API

tugsbayasgalan opened the pull request:
Differential Revision: D45279206  

cc @soumith @voznesenskym @penguinwu @anijain2305 @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @Xia-Weiwen @wenzhe-nrv @jiayisunx @desertfire

gmagogsfm commented on torch/_export/__init__.py:
```diff
@@ -39,7 +49,34 @@
+    dynamic_shapes: bool = True
+    specialize_int: bool = True
+    allow_rnn: bool = True
+    verbose: bool = True
```
Why should this default to True?

tugsbayasgalan committed:
Add private _export API (#99992)

Summary:
Pull Request resolved: https://github.com/pytorch/pytorch/pull/99992

title

Test Plan: CI

Reviewed By: angelayi

Differential Revision: D45279206

fbshipit-source-id: ce5d6f4102638571ec14933b193ae2037fe9c0f0

tugsbayasgalan commented:
@pytorchbot merge -f "Landed internally"

izaitsevfb commented:
@pytorchbot merge -f "Landed internally"
